docs/CHANGELOG/changelog-0.9.x.md

```
@@ -328,3 +328,43 @@ kubernetes:
+  # GPU machine type name.
+  serverType: g4dn.xlarge
+  machineSpec:
+    # explicitly specify how many GPU's the instance type provides.
+    nvidiaGpu: 1
+  image: ami-07eef52105e8a2059
+```
+
+- Add support for OpenStack provider, with the main aim of supporting the openstack offering from [OVH](https://www.ovhcloud.com/en/) [#1857](https://github.com/berops/claudie/pull/1857)
+  It is now possible to use an openstack provider within the InputManifest.
+  The support for openstack has been added in the `v0.9.14` version of the claudie templates.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)